data_prep/scotland/geojson_to_osmium_extracts.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import json
+
+
+# This tool takes a GeoJSON with many features, and prepares Osmium to extract
+# a boundary for each one. The output filenames are based on a feature's `kind`
```

Comment:
So it doesn't actually *run* osmium to do the extracts, but it generates a config for osmium to do the extracts later. Right?

Reply:
Correct. I added d2e6c8504bedf88d1afcdd6feaca00074cb9a618 to clarify a bit. This script generates a .geojson file per boundary an osmium config to batch a few boundaries together. Then the shell script runs through all of those. It's mostly necessary to speed up the extraction by working in batches.